packages/diagram/src/module/diagrams/content/uml/sequence/types.ts

```diff
@@ -0,0 +1,102 @@
+import {
+    booleanType,
+    functionType,
+    listType,
+    literal,
+    numberType,
+    objectType,
+    optional,
+    or,
+    stringType
+} from "@hylimo/core";
+import { canvasContentType, canvasPointType } from "../../../../base/types.js";
+
+// Contains all Types used within sequence diagrams
+
+export const eventType = objectType(
```

Comment:
I think it would be better here to have a doc comment for each of the constants, and then list and explain the fields in this doc comment, to not have so many inline comments